src/test/java/blackjack/domain/user/status/HandTest.java

```diff
@@ -0,0 +1,81 @@
+package blackjack.domain.user.status;
+
+import blackjack.domain.card.Card;
+import blackjack.domain.card.Cards;
+import blackjack.domain.card.painting.Suit;
+import blackjack.domain.card.painting.Value;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class HandTest {
+    @BeforeEach
+    void setUp() {
+        List<Card> cards = Arrays.asList(
+                new Card(Suit.CLOVER, Value.JACK), new Card(Suit.DIAMOND, Value.FOUR),
+                new Card(Suit.HEART, Value.ACE));
+        hand = new Hand(new Cards(cards));
+    }
+
+    @DisplayName("손에 들고 있는 카드 목록을 구한다")
+    @Test
+    void testGetCards() {
+        //given
+        //when
+        List<Card> cards = hand.getCards();
+        //then
+        assertThat(cards.size()).isEqualTo(3);
```

Comment:
```suggestion
        assertThat(cards).hasSize(3);
```

리스트 형태의 size check는 이런 식으로도 할 수 있습니다 :)